webrtc.html

```diff
@@ -8852,6 +8854,34 @@ <h2 id="sec.stats-model">Statistics Model</h2>
       [[!WEBRTC-STATS]].</p>
     </section>
     <section>
+      <h3>The stats selection algorithm</h3>
+      <p>
+	The <dfn>stats selection algorithm</dfn> is as follows:
+	<ul>
+	  <li>If the selector is a MediaStreamTrack, return the following objects:
+	    <ul>
+	      <li>
+		The RTCMediaStreamStats corresponding to the selector.
+	      </li>
+	      <li>
+		If the selector track is inbound, the RTCInboundRTPStreamStats corresponding to the selector.
```

Comment:
(Unless you count a complete GetStats call as such API.) I don't think an API for looking up specific references works anyway, because a separate call occurs at a later time, at which these relationships could have changed.

I agree avoiding broken references would be nice to achieve if possible. Precluding broken references and other tracks would mean we can't say which stream a track belongs to, because other tracks may belong to that stream. On the other hand, including those tracks might be fine. Tracks don't reference things (other things reference tracks). We would have to be careful about not introducing the wrong references if we have selectors like this. This might be fine, but what if we want other types of selectors, or other references in the future? (Its possible I'm prematurely worrying.)

We could always not include references that we don't want to be followable, saying only relevant references are included, but that could be misleading.